keras/backend/theano_backend.py

```diff
@@ -642,6 +700,67 @@ def conv2d(x, kernel, strides=(1, 1), border_mode='valid', dim_ordering='th',
     return conv_out
 
 
+def conv3d(x, kernel, strides=(1, 1, 1), border_mode='valid', dim_ordering='th',
+           image_shape=None, filter_shape=None):
+    '''
+    Run on cuDNN if available.
```

Comment:
I think you are talking about the conv2d in theano_backend.py used by
Convolution2D, when I say it should be updated is because it's using an
outdated interface, it's not nnet.conv2d.
On Fri, Feb 19, 2016 at 03:47 Joaquín Ruales <notifications@github.com>
wrote:

> In keras/backend/theano_backend.py
> <https://github.com/fchollet/keras/pull/1623#discussion_r53417092>:
>
> > @@ -642,6 +700,67 @@ def conv2d(x, kernel, strides=(1, 1), border_mode='valid', dim_ordering='th',
> >      return conv_out
> >
> >
> > +def conv3d(x, kernel, strides=(1, 1, 1), border_mode='valid', dim_ordering='th',
> > +           image_shape=None, filter_shape=None):
> > +    '''
> > +    Run on cuDNN if available.
>
> conv3d2d does use the new version nnet.conv2d so it should be fine
> https://github.com/Theano/Theano/blob/master/theano/tensor/nnet/conv3d2d.py#L244
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fchollet/keras/pull/1623/files#r53417092>.
>